girder_annotation/girder_large_image_annotation/web_client/views/annotationListWidget.js

```diff
@@ -182,24 +193,109 @@ const AnnotationListWidget = View.extend({
         this.$el.find('th.g-annotation-toggle i').toggleClass('icon-eye', !anyOn).toggleClass('icon-eye-off', !!anyOn);
     },
 
-    _deleteAnnotation(evt) {
+    _selectAnnotation(evt) {
+        // Prevent event from bubbling up to the row click handler
+        // that toggles the annotation on and off
+        evt.stopPropagation();
+
         const $el = $(evt.currentTarget);
         const id = $el.parents('.g-annotation-row').data('annotationId');
+        const allChecks = this.$el.find('.g-annotation-select input[type=checkbox]');
+        const anySelected = this.$('.g-annotation-select input:checked').length > 0;
+
+        this.$('thead .g-annotation-download-selected, thead .g-annotation-delete, thead .g-annotation-permissions')
+            .prop('disabled', !anySelected)
+            .toggleClass('disabled', !anySelected)
+            .css('color', anySelected ? '' : 'grey');
         if (!id) {
-            confirm({
-                text: `Are you sure you want to delete <b>ALL</b> annotations?`,
-                escapedHtml: true,
-                yesText: 'Delete',
-                confirmCallback: () => {
-                    restRequest({
-                        url: `annotation/item/${this.model.id}`,
-                        method: 'DELETE'
-                    }).done(() => {
-                        this.collection.fetch(null, true);
+            if ($el.find('input').is(':checked')) {
```

Comment:
`is(':checked')` only checks the first element, so I'm not sure what this is actually doing (at least rechecking all the individual boxes doesn't turn the all-box back on.  I probably would have done `allChecks.prop('checked', $el.find('.g-annotation-row .g-annotation-select input:not(:checked)').length === 0)` instead of the pair of if statements (I didn't test my code).